elements-fun/src/transaction.rs

```diff
@@ -23,34 +23,196 @@ use bitcoin::blockdata::script::{Instruction, Script};
 use bitcoin::hashes::Hash;
 use bitcoin::{self, Txid, VarInt};
 
-use crate::confidential;
-use crate::encode::{self, Decodable, Encodable};
+use crate::confidential::AssetCommitment;
+use crate::confidential::NonceCommitment;
+use crate::confidential::ValueCommitment;
+use crate::encode::{self, Decodable, Encodable, Error};
 use crate::issuance::AssetId;
 
+// TODO: Is `Default` really useful here?
```

Comment:
:question: Why not? A null asset issuance is what I would expect when I need the rest of a transaction to be default.